client/state/selectors/test/get-follow-count.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getFollowCount
+} from '../';
+
+describe( 'getFollowCount()', () => {
+	it( 'should return zero if nothing has been followed', () => {
+		const count = getFollowCount( {
+			reader: {
+				follows: {
+					items: {},
+				}
+			}
+		} );
+
+		expect( count ).to.eq( 0 );
+	} );
+
+	it( 'should return the count if something has been followed', () => {
```

Comment:
Test for > 1 added! Reducer looks okay but I've fixed the selector as per your other comment: https://github.com/Automattic/wp-calypso/pull/10777#discussion_r97580896.